apps/landing-page/content/create-whatsapp-chatbot.mdx

```diff
@@ -0,0 +1,295 @@
+
+### Incorporating Rich Media
+
+WhatsApp supports various rich media formats. This allows you to enhance your chatbot's visual appeal and engagement. Strategically incorporate:
+
+- Images
+- GIFs
+- Videos to illustrate key points, add visual interest, or inject fun and personality.
+
+However, keep in mind that WhatsApp has some limitations regarding media formats. For instance:
+
+- GIF and SVG image files are not supported.
+- Only .mp4 videos are compatible.
+
+Typebot's documentation provides a list of supported media types to ensure your content displays correctly.
+
+### Handling User Input and Providing Relevant Responses
+
+One of the most crucial aspects of an engaging chatbot is its ability to understand and respond to user input effectively. Typebot's input fields and conditional logic features allow you to gather information from users and tailor your chatbot's responses accordingly.
+
+You can use input fields to gather user preferences or demographic info. Then, use conditional logic to offer personalized recommendations or content based on their answers.
+
+### Limitations of WhatsApp Chatbots
+
+While WhatsApp chatbots offer a powerful and convenient way to engage with your audience, it's important to be aware of their limitations.
+
+- WhatsApp only allows you to display three buttons at a time. Typebot will add "..." messages to display additional buttons.
+- Certain Typebot blocks are incompatible with the WhatsApp environment, such as:
+  - Payment input block
+  - Chatwoot block
+  - Script block
+  - Google Analytics block
+  - Meta Pixel blocks
+  - [Cal.com](http://cal.com/) block
+
+These blocks will be skipped if present in your chatbot's flow.
+
+By understanding and working within these limitations, you can create a seamless and enjoyable experience for your WhatsApp chatbot users.
+
+With these design principles and strategies in mind, you're well on your way to creating a WhatsApp chatbot that provides value and leaves a lasting impression.
+
+## Advanced WhatsApp Chatbot Features
+
+As you gain more experience with building and optimizing your WhatsApp chatbot, you may want to explore advanced features and capabilities. Typebot offers powerful tools and integrations to create sophisticated and personalized chatbots.
+
+### Contact Information and System Variables
+
+One powerful aspect of WhatsApp is automatically capturing user contact information, such as their name and phone number.
+
+Typebot makes it easy to leverage this data within your chatbot flow using dedicated system variables.
+
+For example, you can use the `{{waba.name}}` variable to personalize your chatbot's greetings and responses with the user's name. This simple touch can make the experience feel more human and tailored to the individual.
+
+Similarly, the `{{waba.phone}}` variable can pre-populate input fields or integrate with systems that require the user's phone number, streamlining the overall experience.
```

Comment:
This is not how it works (see https://docs.typebot.io/deploy/whatsapp/overview#contact-information), the user need to add a set variable block with the WhatsApp values